pkg/controllers/user/helm/common/common.go

```diff
@@ -49,6 +54,34 @@ func ParseExternalID(externalID string) (string, string, error) {
 	return strings.Join([]string{catalog, template, version}, "-"), templateVersionNamespace, nil
 }
 
+func IsSystemCatalog(obj *v3.App) {
```

Comment:
Please name the function with a more readable name. Like `InjectDefaultRegistry`

Reply:
Updated!